src/user/user.model.ts

```diff
@@ -7,12 +7,36 @@ const userSchema = new mongoose.Schema<IUser>({
     required: true,
     unique: true,
     lowercase: true,
+    IsEmail: true,
+    IsString: true, 
+    maxlength: 100,
+    
   },
   password: {
     type: String,
     required: true,
     select: false,
+    minlength: 6,
+    maxlength: 100,
   },
+  programName:{
+    type: String,
+    required: true,
+    IsString: true,
+    maxlength: 100,
+  },
+  group:{
+    type: Number,
+    required: true,
+    IsNumber: true,
+  },
+  subGroup:{
+    type: Number,
+    required: true,
+    IsNumber: true, 
+  }
 });
 
+// bus galima kazkada anglu dadet
```

Comment:
change this comment to be:
"// TODO: add english group and subGroup", the use of "TODO", will help us find what we have not implemented more easily in the future